web/app/scripts/views/record/list-controller.js

```diff
@@ -37,8 +37,8 @@
 
             return RecordSchemaState.get(currentSchemaId)
                 .then(function(recordSchema) {
-                    ctl.recordSchema = recordSchema;
-                });
+                    return ctl.recordSchema = recordSchema;
```

Comment:
This works, but it makes jshint nervous:
<pre>app/scripts/views/record/list-controller.js
  line 40  col 46  Did you mean to return a conditional instead of an assignment?

  ⚠  1 warning

Warning: Task "jshint:all" failed. Use --force to continue.

Aborted due to warnings.</pre>

+1 after placating jshint with a separate return statement.